src/descriptors.rs

```diff
@@ -49,6 +53,12 @@ impl<'a> ClassName<'a> {
             .fold(0, |sum, segment| sum + segment.name.len() + 1)
     }
 }
+impl<'a> fmt::Display for ClassName<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
+        let segments : Vec<Cow<'a, str>> = self.segments.iter().map(|s| s.name.clone()).collect();
+        write!(f, "{}", segments.join("/"))
```

Comment:
Joining on `/` isn't quite perfect here as I think the segments can also separated by `;`. Thoughts?

Reply:
Oh I missed the `;` was being added at the callsite where you serialize the `L{};` - so your original implementation was right, don't need the extra `;`, sorry!

Reply:
Ah Cool. #56 fixes that!

Reply:
Thanks, I've merged that and some test changes too. Published everything as 0.8.1. Cheers!